README.md

```diff
@@ -1,10 +1,35 @@
-# @crowdstrike/ember-toucan-core
+# Toucan Core
+
+<p align="center">
+  <a href="https://github.com/crowdstrike/ember-toucan-core">
+    <img src="https://github.com/CrowdStrike/ember-toucan-core/blob/main/.github/cs-logo.png?raw=true" alt="CrowdStrike logo" width="300" />
```

Comment:
**NOTE:** If you [view this file rendered](https://github.com/CrowdStrike/ember-toucan-core/blob/f2421be1b7624d7f1b11e930ec1fe192396dfc4c/README.md), you will notice the image is not displayed. That's because the image is not yet on `main`. Once this merges, it should be good to go 👍. A bit of a chicken/egg problem.

To test this out you could manually edit the <img src attribute to src="https://github.com/CrowdStrike/ember-toucan-core/blob/update-readme/.github/cs-logo.png?raw=true" and you should see:

<img width="1016" alt="246419556-fc467667-aa7e-40f5-8e75-0fcd9c012d5e" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/c374d40a-ba08-4daa-a291-da947e3a923b">